.github/workflows/package.yml

```diff
@@ -37,3 +37,24 @@ jobs:
         with:
           name: artifact-${{ matrix.image }}
           path: release/
+
+  plugin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js LTS
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: 'npm'
+      - name: Authenticate with npm registry
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: generate contents
+        run: |
+          npm install
+          npm run build:plugin
+      - name: publish
+        run: npm publish
```

Comment:
not sure if this should be done via the release plugin, which also maintains changelist

Reply:
We currently manage changelist in https://github.com/appium/appium-inspector/releases manually, so this doesn't affect the existing changelist management